daemon/api_themes.go

```diff
@@ -227,7 +227,7 @@ func installThemes(c *Command, r *http.Request, user *auth.UserState) Response {
 	if user != nil {
 		userID = user.ID
 	}
-	installed, tasksets, err := snapstateInstallMany(st, toInstall, userID)
+	installed, tasksets, err := snapstateInstallMany(st, toInstall, userID, false)
```

Comment:
here and at other call sites:
```suggestion
	const transactional = false
	installed, tasksets, err := snapstateInstallMany(st, toInstall, userID, transactional)
```
unless we come up with something more descriptive in place of the bool argument